build.gradle

```diff
@@ -23,12 +23,14 @@ apply plugin: "com.gradle.plugin-publish"
 
 repositories {
     jcenter()
+    maven { url 'https://jitpack.io' }
 }
 
 dependencies {
     compile gradleApi()
     compile 'org.codehaus.groovy:groovy-all:2.4.10'
-    compile 'com.github.groovy-wslite:groovy-wslite:1.1.3'
+    // compile 'com.github.groovy-wslite:groovy-wslite:1.1.3'
```

Comment:
I used Jitpack as a temporary solution until the new version of groovy-wslite is officially released/published (it seems to take years) . 

> whereas I had instability issues with 2.x. 

I had no problems with 2.x and Java 8, but there were some with 1.x .